### ISSUE TITLE
`jbrowse upgrade` does not clean up old worker files

### ISSUE THREAD

**Issue by ilaydabozan** [GMOD/jbrowse-components#2337]
I don't know if this is expected behaviour, but when `jbrowse upgrade` is run, many copies of the following three files are produced, while the old set of many files remains with the different hashes.

```
worker.js
worker.js.LICENSE.txt
worker.js.map 
```

**To Reproduce**

Run `jbrowse upgrade`

**Screenshots**

here is what the files are:

```
-rw-rw-r--    1 ubuntu ubuntu  175644 Sep 16 13:10 0.ff8da6869c2040672276.worker.js
-rw-rw-r--    1 ubuntu ubuntu     173 Sep 16 13:10 0.ff8da6869c2040672276.worker.js.LICENSE.txt
-rw-rw-r--    1 ubuntu ubuntu  398593 Sep 16 13:10 0.ff8da6869c2040672276.worker.js.map
...
...
-rw-rw-r--    1 ubuntu ubuntu    1682 Sep 16 13:10 50.ff8da6869c2040672276.worker.js
-rw-rw-r--    1 ubuntu ubuntu    2914 Sep 16 13:10 50.ff8da6869c2040672276.worker.js.map
-rw-rw-r--    1 ubuntu ubuntu    2701 Sep 16 13:10 51.ff8da6869c2040672276.worker.js
-rw-rw-r--    1 ubuntu ubuntu    5480 Sep 16 13:10 51.ff8da6869c2040672276.worker.js.map
-rw-rw-r--    1 ubuntu ubuntu    6181 Sep 16 13:10 52.ff8da6869c2040672276.worker.js
-rw-rw-r--    1 ubuntu ubuntu   17346 Sep 16 13:10 52.ff8da6869c2040672276.worker.js.map
-rw-rw-r--    1 ubuntu ubuntu    1784 Sep 16 13:10 53.ff8da6869c2040672276.worker.js
-rw-rw-r--    1 ubuntu ubuntu    5239 Sep 16 13:10 53.ff8da6869c2040672276.worker.js.map
```

And only some has the copy of the .LICENSE.txt
```
-rw-rw-r--    1 ubuntu ubuntu     173 Sep 16 13:10 0.ff8da6869c2040672276.worker.js.LICENSE.txt
-rw-rw-r--    1 ubuntu ubuntu     173 Sep 16 13:10 1.ff8da6869c2040672276.worker.js.LICENSE.txt
-rw-rw-r--    1 ubuntu ubuntu     173 Sep 16 13:10 26.ff8da6869c2040672276.worker.js.LICENSE.txt
-rw-rw-r--    1 ubuntu ubuntu     173 Sep 16 13:10 27.ff8da6869c2040672276.worker.js.LICENSE.txt
-rw-rw-r--    1 ubuntu ubuntu     120 Sep 16 13:10 3.ff8da6869c2040672276.worker.js.LICENSE.txt
-rw-rw-r--    1 ubuntu ubuntu    6548 Sep 16 13:10 ff8da6869c2040672276.worker.js.LICENSE.txt
```

When following command is ran to remove the copies:
`rm *.ff8da6869c2040672276.*`
`npx serve .` command works as it's intended with the remaining files:

```
-rw-rw-r--    1 ubuntu ubuntu 2469983 Sep 16 13:10 ff8da6869c2040672276.worker.js
-rw-rw-r--    1 ubuntu ubuntu    6548 Sep 16 13:10 ff8da6869c2040672276.worker.js.LICENSE.txt
-rw-rw-r--    1 ubuntu ubuntu 8747310 Sep 16 13:10 ff8da6869c2040672276.worker.js.map
```

**Version:**
```
@jbrowse/cli/1.0.4 linux-x64 node-v13.14.0
```
### Server details
```
NAME="Ubuntu"
VERSION="18.04.5 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.5 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```

Thank you,
Ilayda

**Comment by garrettjstevens** [comment]
JBrowse does use a lot of worker files. This is because the code is split up and chunks are loaded on demand. For example, when I tried your `rm *.ff8da6869c2040672276.*` command, The initial JBrowse page still loaded, but I couldn't open any tracks since they depend on the removed worker files.

It's a good point, though, that upgrading JBrowse does not clean up old copies of the worker files. That is something we need to address. I'll modify the issue title to reflect that.

**Comment by ilaydabozan** [comment]
Thank you for the answer! It's much more clear now 👍

**Comment by cmdcolin** [comment]
in the next version we will have a --clean argument to `jbrowse upgrade` to help this :)

**Comment by cmdcolin** [comment]
released in 1.6.6
